oluwajuwon/Fast-Food-Fast
# #163442164 Users should be able to delete their order(s)

### What does this pull request do?
This pull request creates a feature for a user to delete his/her order

### Description of the Task to be completed?
A user should be able to delete his/her order from a list of orders

### How should this be manually tested?
Users can test it by:
 - Install git
 - Open up the command line
 - Run `git clone https://github.com/oluwajuwon/Fast-Food-Fast.git` to clone the repository. 
 - Run `npm install` to install all package dependencies
 - Run `npm run start` to start the app
 - Visit http://localhost:3000/api/v1/auth/login to login as as user with valid credentials
 - Add the generated token as header 
 - Send a delete request to   http://localhost:3000/api/v1/orders/:id' to see a delete an order

### What are the relevant pivotal tracker stories?
The relevant pivotal story is:
Admin should be able to delete a particular menu item - https://www.pivotaltracker.com/story/show/163442164


### Screenshots
N/A
